common/devdoc/gballoc_design.md

```diff
@@ -0,0 +1,40 @@
+gballoc design
+=====
+
+`gballoc` is a layer that abstrats memory allocators. The need to have and be able to compare multiple memory allocators stems from performance requirements. Not all memory allocators have the same performance and for the sake of being able to compare them gballoc exists.
```

Comment:
>abstrats [](start = 26, length = 8)

abstracts

Reply:
abstracted.

---
In reply to: [457670558](https://github.com/Azure/azure-c-pal/pull/11#discussion_r457670558) [](ancestors = 457670558)